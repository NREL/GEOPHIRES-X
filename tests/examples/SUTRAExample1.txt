GEOPHIRES v3.0 Input File
Created on 11/3/2023
Last modified on 11/3/2023
Reservoir thermal energy storage simulated with SUTRA

This example reads in a SUTRA simulation result and calculate techno-economics of seasonal reservoir thermal energy storage.


***Subsurface technical parameters***
*************************************
Reservoir Model,7,                        			---Reads in SUTRA result for seasonal reservoir thermal energy storage
SUTRA Annual Heat File Name,Examples/annual_heat.csv,			---SUTRA file with heat stored, heat supplied and efficiency for each year
SUTRA Heat Budget File Name,Examples/heat_budget.csv,			---SUTRA file with target heat and simulated heat for each SUTRA time step over lifetime
SUTRA Balance and Storage Well Output File Name,Examples/balance_and_storage_well_output.csv,	---SUTRA file with well flow rate and temperature for each SUTRA time step over lifetime
Number of Production Wells,1,            			---[-]
Number of Injection Wells,1,            			---[-]
Reservoir Depth,0.6,                   				---[km]
Number of Segments,1,                  				---[-]
Surface Temperature,10,						---[deg.C]
Gradient 1,0.01,                          			---[deg.C/km]
Production Well Diameter,7.8,		     			---[inch]
Injection Well Diameter,7.8,					---[inch]
Ramey Production Wellbore Model,0,       			---0 if disabled  1 if enabled
Production Wellbore Temperature Drop,0,				---[deg.C]
Injection Wellbore Temperature Gain,0,   			---[deg.C]
Reservoir Impedance,0.2,					---[GPa*s/m3]

***SURFACE TECHNICAL PARAMETERS***
**********************************
End-Use Option,2,			  			        ---[-] Direct-Use Heat
Circulation Pump Efficiency,.8,	  				---[-] between .1 and 1
Power Plant Type,8,                             ---[-] Reservoir Thermal Energy Storage

***FINANCIAL PARAMETERS***
**************************
Plant Lifetime,30,			  			---[years]
Economic Model,2,			  			---[-] Standard Levelized Cost Model
Fixed Charge Rate,.05,			 			---[-] between 0 and 1
Inflation Rate During Construction,0,   			---[-]
End-Use Efficiency Factor,1,					---[-] End-Use Efficiency

***CAPITAL AND O&M COST PARAMETERS***
*************************************
Well Drilling and Completion Capital Cost Adjustment Factor,1,	---[-] Use built-in correlations
Well Drilling Cost Correlation,1,				---[-] Use built-in correlations
Reservoir Stimulation Capital Cost Adjustment Factor,1,		---[-] Use built-in correlations
Surface Plant Capital Cost Adjustment Factor,1,			---[-] Use built-in correlations
Field Gathering System Capital Cost Adjustment Factor,1,	---[-] Use built-in correlations
Exploration Capital Cost Adjustment Factor,1,			---[-] Use built-in correlations
Wellfield O&M Cost Adjustment Factor,1,				---[-] Use built-in correlations
Surface Plant O&M Cost Adjustment Factor,1,			---[-] Use built-in correlations
Water Cost Adjustment Factor,1,					---[-] Use built-in correlations


***Simulation Parameters***
***************************
Print Output to Console,1,		  			---[-] Should be 0 (don't print results) or 1 (print results)
