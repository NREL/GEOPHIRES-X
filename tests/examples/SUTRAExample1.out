                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.0
 GEOPHIRES Build Date: 2023-11-06
 Simulation Date: 2023-11-06
 Simulation Time:  17:26
 Calculation Time:      1.566 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Direct-Use Heat
      Reservoir Model = SUTRA Model
      Direct-Use heat breakeven price:                     1730.66 cents/kWh
      Number of Production Wells:                             1
      Number of Injection Wells:                              1
      Lifetime Average Well Flow Rate:                     16.2 kg/sec
      Well depth:                                         600.0 meter


                           ***ECONOMIC PARAMETERS***

      Economic Model = Standard Levelized Cost
      Interest Rate:                                          7.00 %
      Accrued financing during construction:                  0.00 %
      Project lifetime:                                      30 yr

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             1
      Number of Injection Wells:                              1
      Well Depth:                                         600.0 meter
      Pump efficiency:                                     80.0
      Lifetime Average Well Flow Rate:                     16.2 kg/sec
      Injection well casing ID:                           0.198 meter
      Production well casing ID:                          0.198 meter


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Storage Well Temperature:                    30.1 degC
      Average Storage Well Temperature:                     6.2 degC
      Minimum Storage Well Temperature:                     4.0 degC
      Maximum Balance Well Temperature:                    30.5 degC
      Average Balance Well Temperature:                    15.3 degC
      Minimum Balance Well Temperature:                    14.0 degC
      Maximum Annual Heat Stored:                          -3.3 GWh
      Average Annual Heat Stored:                          -3.4 GWh
      Minimum Annual Heat Stored:                          -3.4 GWh
      Maximum Annual Heat Supplied:                         2.5 GWh
      Average Annual Heat Supplied:                         2.2 GWh
      Minimum Annual Heat Supplied:                         0.6 GWh
      Average Round-Trip Efficiency:                       64.4 %
      Total Average Pressure Drop:                         35.3 kPa


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Average RTES Heating Production:                     0.25 MW
      Average Auxiliary Heating Production:                0.14 MW
      Average Annual RTES Heating Production:              2.17 GWh/year
      Average Annual Auxiliary Heating Production:         1.21 GWh/year
      Average Annual Total Heating Production:             3.39 GWh/year
      Average Pumping Power:                               2.31 kW
      Average Annual Electricity Use for Pumping:      20261.85 kWh/yr


                          ***CAPITAL COSTS (M$)***

      Drilling and Completion Costs:                       2.42 MUSD
      Drilling and Completion Costs per Well:              1.21 MUSD
      Auxiliary Heater Cost:                               0.21 MUSD
      Pump Cost:                                           0.17 MUSD
      Total Capital Costs:                                 2.81 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

      Average annual auxiliary fuel cost:                 48.59 KUSD/yr
      Average annual pumping cost:                         1.42 KUSD/yr
      Total average annual O&M costs:                     50.01 KUSD/yr
